app/views/stories/tagged_articles/index.html.erb

```diff
@@ -24,7 +24,7 @@
               <button
                 id="user-follow-butt"
                 class="crayons-btn follow-action-button"
-                data-info='{"id":<%= @tag.id %>,"className":"Tag", "name": "<%= @tag.pretty_name || @tag.name %>"}'>
+                data-info='<%= DataInfo.to_json(object: @tag, class_name: "Tag", name: @tag.pretty_name || @tag.name) %>'>
```

Comment:
We may be able to get away with not passing "Tag" but I've seen both a `Tag` object and `ActsAsTaggableOn::Tag` object come through in my copious amounts of outside looking in debugging.